lib/driverProviders/sauce.js

```diff
@@ -5,7 +5,7 @@
  */
 
 var util = require('util'),
-    log = require('../logger.js'),
+    log = require('../logger.js').Logger,
```

Comment:
No, there is another syntax to add an export default at the end of the file, choosing which symbol from the scope should be the default export.
So TypeScript should be able to express the logger.ts file so that the usage sites are unchanged.